lib/cstring.h

```diff
@@ -191,7 +191,7 @@ class cstring {
     std::string_view string_view() const {
         return str ? std::string_view(str) : std::string_view("");
     }
-    explicit operator std::string_view() const { return string_view(); }
+    operator std::string_view() const { return string_view(); }
```

Comment:
Would add a comment here why this shouldn't be explicit.

Reply:
This was already discussed in https://github.com/p4lang/p4c/issues/4688. But essentially: we are going to make interface similar to `std::string` essentially allowing `string_view` into corresponding cstring-backed storage. This would fix lots of `cstring` misuse.